### DISCUSSION TITLE
服务器端是否需要修改配置以满足 XTLS Vision ？

### DISCUSSION THREAD

**Discussion by DanielChen008** [XTLS/Xray-core#1695] [Q&A]
私有 Server 运行很久很稳定，看到 1.7.5 开始会支持新的流控 XTLS Vision。
想确认一下，服务器端的配置是否有需要调整的地方？
好提前准备，因为涉及到服务器、软路由固件的重新编译、移动客户端都要调整。

**Comment by chika0801** [comment]
服务器配置格式你参考模板，改动的地方只有几个。路由器插件方面ssrp+和passwall已经更新了支持了vision。
安卓最新版本的V2rayNG都支持。IOS建议你开hysteria或tuic用。
